package.json

```diff
@@ -107,9 +107,6 @@
   "engines": {
     "node": "^6.x.x || ^7.x.x"
   },
-  "jest": {
-    "notify": true
```

Comment:
The issue was "for each notif, a new process was created". I need to find the issue again and see if it hasn't been fixed since.